src/cli/cmd/login/mod.rs

```diff
@@ -40,8 +49,60 @@ impl CommandExecute for LoginSubcommand {
     }
 }
 
+pub async fn dnixd_uds() -> color_eyre::Result<SendRequest<axum::body::Body>> {
+    let dnixd_state_dir = Path::new(&DETERMINATE_STATE_DIR);
+    let dnixd_uds_socket_path: PathBuf = dnixd_state_dir.join(DETERMINATE_NIXD_SOCKET_NAME);
+
+    let stream = TokioIo::new(UnixStream::connect(dnixd_uds_socket_path).await?);
+    let (mut sender, conn): (SendRequest<Body>, _) =
+        hyper::client::conn::http1::handshake(stream).await?;
+
+    // NOTE(colemickens): for now we just drop the joinhandle and let it keep running
+    let _join_handle = tokio::task::spawn(async move {
+        if let Err(err) = conn.await {
+            tracing::error!("Connection failed: {:?}", err);
+        }
+    });
+
+    let request = http::Request::builder()
+        .method(Method::GET)
+        .uri("http://localhost/info")
+        .body(axum::body::Body::empty())?;
+
+    let response = sender.send_request(request).await?;
+
+    assert_eq!(response.status(), StatusCode::OK);
```

Comment:
Why are we asserting it's equal and then checking-and-erroring right after?
```suggestion
```

Reply:
ah, copy pasted code, wrote the better version, forgot to delete the assert. fixed, pushed.